.flake8

```diff
@@ -12,6 +12,7 @@ exclude =
     ansible/*,
     docs/*,
     .cache/*
+    utilities/tests/*
```

Comment:
nit - how about to rename the folder to `utilities/unittests/` just to have it different from common tests

Reply:
done